arviz/plots/backends/matplotlib/traceplot.py

```diff
@@ -67,8 +70,12 @@ def plot_trace(
         If True adds a rugplot. Defaults to False. Ignored for 2D KDE. Only affects continuous
         variables.
     lines : tuple or list
-        list of tuple of (var_name, {'coord': selection}, [line_positions]) to be overplotted as
+        List of tuple of (var_name, {'coord': selection}, [line_positions]) to be overplotted as
         vertical lines on the density and horizontal lines on the trace.
+    circ_var_names : list
+        List of circular variables to account for when plotting KDE.
+    degrees : bool
```

Comment:
maybe this should be "circ_var_units" and defaults to radians. 

Additionally. I am ok with having degrees or radians for now. But we may want to contemplate other options like allowing negative values, like going from -180 to 180 instead of from 0 to 360 (and the same for radians).